package.json

```diff
@@ -13,14 +14,16 @@
     "prop-types": "^15.8.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-redux": "^8.1.2",
     "react-router-dom": "^6.15.0",
     "react-scripts": "5.0.1",
     "react-test-renderer": "^18.2.0",
     "sort-by": "^1.2.0",
+    "uuid": "^9.0.0",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "DISABLE_ESLINT_PLUGIN=true react-scripts start",
```

Comment:
- [ ] I kindly suggest you remove the `"DISABLE_ESLINT_PLUGIN=true` as it prevents your code from being run on the browser once `npm start` is used 👍